README.md

```diff
@@ -30,7 +30,7 @@ Chrome OS to the Dev Channel](https://support.google.com/chromebook/answer/10869
 and then install Automation Inspector as an app (insert link here).
 * Chrome tabs: you can inspect individual browser tabs on any OS. To do this,
 [install Google Chrome Canary](https://www.google.com/chrome/browser/canary.html) and
-then install Automation Inspector as an extension (insert link here).
+then install [Automation Inspector extension](https://chrome.google.com/webstore/detail/automation-inspector-exte/magknkflhkbbjaalinnipadaihkbghoc).
```

Comment:
Not very sure `https://chrome.google.com/webstore/detail/automation-inspector-exte/magknkflhkbbjaalinnipadaihkbghoc` is a permanent link right now...